model/command_response_test.go

```diff
@@ -19,6 +19,9 @@ func TestCommandResponseFromHTTPBody(t *testing.T) {
 		{"", "foo", "foo"},
 		{"text/plain", "foo", "foo"},
 		{"application/json", `{"text": "foo"}`, "foo"},
+		{"x-www-form-urlencoded", `{"text": "foo"}`, "foo"},
```

Comment:
I'm thinking the provided body should be made to resemble the URL encoded format here, to have a realistic test

Reply:
The core of the new logic is to always try to parse JSON, despite the Content-Type given. This test case was meant exactly to test this path. 

Or, do you mean that the JSON string should be url encoded?

Reply:
I understand what you mean, though I guess it just seemed unrealistic to me that the request would have `x-www-form-urlencoded` as the header if the data is not `x-www-form-urlencoded`. I understand the point of the PR is to favor JSON though so it makes sense.